zerver/webhooks/sonarqube/fixtures/success.json

```diff
@@ -0,0 +1,64 @@
+{
+  "serverUrl": "http://localhost:9000",
+  "taskId": "AXgTFfXRZCzhMRNj54bo",
+  "status": "SUCCESS",
+  "analysedAt": "2021-03-08T18:25:04+0000",
+  "changedAt": "2021-03-08T18:25:04+0000",
+  "project": {
+    "key": "test-sonar",
+    "name": "test-sonar",
+    "url": "http://localhost:9000/dashboard?id=test-sonar"
+  },
+  "branch": {
+    "name": "master",
```

Comment:
Can you work the branch name into the message?  I think it's likely to be important for any project that has multiple branches.

Reply:
Ok.